### PR TITLE
fix(onboarding): links quickstart

### PR DESCRIPTION
**Pull request by PixelDust22**
- Fixed `onClickSetupLater` handler for first event footer
- Added pointer style to onboarding card
- Changed copy of `Next Platform` button.

### CODE REVIEW

File: static/app/components/onboardingWizard/onboardingCard.tsx [hunk 1]
```diff
@@ -174,6 +174,7 @@ const StyledIconClose = styled(IconClose)`
   position: absolute;
   right: ${space(1.5)};
   top: ${space(1.5)};
+  cursor: pointer;
```
Comment (evanpurkhiser) [inline static/app/components/onboardingWizard/onboardingCard.tsx:177]: Can we switch this to a button? We shouldn't be adding `onClick` to svg icons

Comment (evanpurkhiser) [reply static/app/components/onboardingWizard/onboardingCard.tsx:177]: probably want `<CloseButton borderless size="zero" icon={<IconClose color="gray300" />} />` and then the `CloseButton` has some of these styles

Comment (Neo-Zhixing) [reply static/app/components/onboardingWizard/onboardingCard.tsx:177]: Want me to change the other one here as well? https://github.com/getsentry/sentry/blame/7c90d3ea2fa41326738f1b13821b010cf040bafa/static/app/components/onboardingWizard/task.tsx#L127

Comment (evanpurkhiser) [reply static/app/components/onboardingWizard/onboardingCard.tsx:177]: Yeah, let's do it in a sep PR

Comment (evanpurkhiser) [reply static/app/components/onboardingWizard/onboardingCard.tsx:177]: Will let someone else review the rest of this PR

**Review by scefali — APPROVED** [review]

**Pull request merged**